hugo/content/docs/cluster/getting-started-net.md

```diff
@@ -235,7 +235,7 @@ To avoid confusion, in this tutorial we'll refer to virtual actors as grains.
 To recap:
 1. Grains are essentially actors, meaning they will process messages one at a time.
 1. Grains are not explicitly crated (activated). Instead, they are created when they receive the first message.
-1. Each grain lives in _one_ of the cluster members.
+1. Each grain lives in _one_ of the cluster nodes.
```

Comment:
Revert, we explicitly use the word "member" for cluster members.
A cluster can have members or clients. "node" is ambiguous.